hazelcast/src/main/java/com/hazelcast/internal/probing/CompressingProbeRenderer.java

```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.internal.probing;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.EOFException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.zip.Deflater;
+import java.util.zip.DeflaterOutputStream;
+import java.util.zip.InflaterInputStream;
+
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+
+public final class CompressingProbeRenderer implements ProbeRenderer {
```

Comment:
should this be part of the probing API? It is an implementation detail for MC so should be placed there for the time being.

Reply:
No. I am not ready with the sending part. I just put it there in the early stages when I wanted to test it.